core/common/src/main/java/tachyon/security/login/AppLoginModule.java

```diff
@@ -141,4 +142,36 @@ public boolean logout() throws LoginException {
 
     return true;
   }
+
+  /**
+   * A callback handler for {@link AppLoginModule}.
+   */
+  @NotThreadSafe
+  public static final class AppCallbackHandler implements CallbackHandler {
```

Comment:
I move it here because it is always used to work together with this particular `AppLoginModule`, and makes no sense to be used in other places.